src/ff/field.rs

```diff
@@ -102,6 +110,35 @@ pub trait Field:
     }
 }
 
+pub trait FieldTypeStr {
```

Comment:
I am not sure how it would play with `FieldType` enum that we would need to support different field types used in queries.